plugin.video.watchbox/resources/lib/api.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Watchbox
+# Copyright (C) 2017 MrKrabat
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from os.path import join
+from cgi import parse_header
+try:
```

Comment:
You could use `sys.version_info` to detect py2 vs py3. It is more elegant than blinding relying on try/except. Just a suggestion though